literature.bib

```diff
@@ -5963,6 +6005,65 @@ @article{BCKW:Dagstuhl19
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2018 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{BON+:ICSME18,
+  author = {John Businge and Moses Openja and Sarah Nadi and Engineer Bainomugisha and Thorsten Berger},
+  title = {{Clone-Based Variability Management in the Android Ecosystem}},
+  booktitle = ICSME,
+  pages = {625--634},
+  publisher = IEEE,
+  address = Piscataway,
+  month = NOV,
+  location = {Madrid, Spain},
+  year = 2018,
+  doi = {10.1109/ICSME.2018.00072},
+}
+
+@inproceedings{SPC:SPLC18,
+  author = {Sree-Kumar, Anjali and Planas, Elena and Claris\'{o}, Robert},
+  title = {{Extracting Software Product Line Feature Models From Natural Language Specifications}},
+  year = 2018,
+  isbn = {9781450364645},
+  publisher = ACM,
+  address = NY,
+  booktitle = SPLC,
+  pages = {43--53},
+  numpages = {11},
+  doi = {10.1145/3233027.3233029},
+  keywords = {NLTK, feature model extraction, natural language processing, requirements engineering, software product line},
+  location = Gothenburg,
+}
+
+@inproceedings{CHN:VaMoS18,
+  author = {Carbonnel, Jessie and Huchard, Marianne and Nebut, Cl\'{e}mentine},
+  title = {{Towards the Extraction of Variability Information to Assist Variability Modelling of Complex Product Lines}},
+  booktitle = VaMoS,
+  year = 2018,
+  month = FEB,
+  publisher = ACM,
+  address = NY,
+  doi = {10.1145/3168365.3168378},
+  pages = {113--120},
+  numpages = {8},
+  keywords = {Variability Extraction, Software Product Line, Reverse Engineering},
+  isbn = {9781450353984},
+  location = {Madrid, Spain},
+}
+
+@inproceedings{MTZ:SPLC18,
+  author = {Martinez, Jabier and T\"{e}rnava, Xhevahire and Ziadi, Tewfik},
+  title = {{Software Product Line Extraction From Variability-Rich Systems: The Robocode Case Study}},
+  booktitle = SPLC,
+  isbn = {9781450364645},
+  publisher = ACM,
+  address = NY,
+  doi = {10.1145/3233027.3233038},
+  pages = {132--142},
+  numpages = {11},
+  keywords = {education, extractive software product line adoption, reverse-engineering, robocode, software product lines},
```

Comment:
remove keywords?

Reply:
similar as numpages - do we use them for anything? I prefer to maintain my own tags instead, reduces cluttering. maybe we should create a policy for that in the SoftVarE meeting?